.gitignore

```diff
@@ -50,7 +50,7 @@ robo.phar
 npm-debug.log
 
 # Build config
-gulp-config.json
+build/gulp-config.json
```

Comment:
Yeah I don't really know how that got changed. I'm guessing it is something that PHPStorm did while I was trying to get gulp working. I'll revert the changes and send another PR this morning. =^D